components/org.wso2.carbon.identity.governance/src/main/java/org/wso2/carbon/identity/governance/listener/IdentityStoreEventListener.java

```diff
@@ -62,6 +62,7 @@ public class IdentityStoreEventListener extends AbstractIdentityUserOperationEve
     private boolean isHybridDataStoreEnable = false;
     private static final String INVALID_OPERATION = "InvalidOperation";
     private static final String USER_IDENTITY_CLAIMS = "UserIdentityClaims";
+    public static final String STORE_IDENTITY_CLAIMS = "StoreIdentityClaims";
```

Comment:
Don't we need to add this to the user store properties? For any OOTB user store that is being created?

Reply:
No, incase if the property is missing `isStoreIdentityClaimsInUserStore` method return false as bydefault value

Reply:
Any way will add this constant to the kernal level later and will use it here after a kernal release.

Reply:
Its ok this is returning false. But we need to add this config to the userstore configs when we create one. Otherwise, we need to send custom APIs to set this value to true by adding the config to the userstore properties file.